introspection-engine/introspection-engine-tests/src/test_api.rs

```diff
@@ -81,12 +81,18 @@ impl TestApi {
 
     pub async fn introspect(&self) -> Result<String> {
         let introspection_result = self.test_introspect_internal(Datamodel::new()).await?;
+
         Ok(datamodel::render_datamodel_and_config_to_string(
             &introspection_result.data_model,
             &self.configuration(),
         ))
     }
 
+    pub async fn introspect_dml(&self) -> Result<String> {
+        let introspection_result = self.test_introspect_internal(Datamodel::new()).await?;
```

Comment:
I still don't like that we're passing in an empty datamodel here. That would not work in the rpc call. I'd love to have the generator and datasource in here to be sure it does the same as the CLI calls.

Reply:
It's another pass we should do later on. If we call `SqlIntrospectionConnector` directly we don't really do anything with the `DataModel`.